### DISCUSSION TITLE
Website works fine locally but after hosting it through Github, the css does not work

### DISCUSSION THREAD

**Discussion by meetnandu05** [community/community#22808] [Other Feature Feedback, Questions, & Ideas]
<p>Hi everyone. I have created my first website and tried hosting it.</p>
<p>The issue I am facing is, the website works properly on my local system. Even after I pull the files from my github repo, it works fine. But on the actual website all the javascript and css is gone. The website is - <a href="http://www.sampurnagraphics.com" rel="noopener nofollow ugc">www.sampurnagraphics.com</a><br>
and the github repo is - <a href="https://github.com/meetnandu05/meetnandu05.github.io" rel="noopener nofollow ugc">https://github.com/meetnandu05/meetnandu05.github.io</a></p>
<p>It would be great if anyone helps me. Thanks!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/meetnandu05">@meetnandu05</a>, welcome to the GitHub Support Community!</p>
<p>Looking in the development console for your site it seems like there’s a lot of 403 (Forbidden) errors when loading your site’s assets. It seems like these assets are being loaded from an external domain <code>https://tympanus.net/</code> which is likely blocking your use of these files.</p>
<p>You’ll either need to find an external host that will allow you to reference these files or upload them directly to your GitHub repository and update the links to point to these files instead.</p>
